css-four-column-layout/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@300;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>CSS Four Column Layout</title>
+  </head>
+  <body>
+    <div class="container background-color margin-top">
+      <div class="row">
+        <div class="column-full">
+          <h4 class="text-center">How It Works</h4>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-full text-center">
+          <img class="img-phone" src="images/screen-1.png" alt="phone">
+          <img class="img-phone" src="images/screen-2.png" alt="phone">
+          <img class="img-phone" src="images/screen-3.png" alt="phone">
+          <img class="img-phone" src="images/screen-4.png" alt="phone">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-full space-evenly margin rel">
+          <img class="img-icon padding" src="images/switch-icon.png" alt="App Switching">
+          <img class="img-icon" src="images/filter-icon.png" alt="trip filtering">
+          <img class="img-icon" src="images/slider-icon.png" alt="service filters">
+          <img class="img-icon" src="images/bars-icon.png" alt="mileage">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-full space-evenly margin rel-h">
+          <h6>App Switching</h6>
+          <h6>Trip Filtering</h6>
+          <h6>Service Filter</h6>
+          <h6>Status + Mileage</h6>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-full space-evenly margin-p rel-p">
+          <p class="font"> Toggles all your apps on and off.</p>
+          <p class="font"> Evaluates your trip requests</p>
+          <p class="font"> Accepts only your ideal trips</p>
+          <p class="font"> Tracts your trips accross all apps</p>
+        </div>
+      </div>
+    </div>
```

Comment:
All of this content is related, so it should not be separated into individual rows. You want one row with four columns, each of which holds the phone image, icon, and text associated with that image and icon.